papis/docmatcher.py

```diff
@@ -28,8 +28,21 @@ class DocMatcher(object):
     """
     search = ""  # type: str
     parsed_search = []  # type: List[Any]
-    doc_format = '{%s[DOC_KEY]}' % (papis.config.getstring('format-doc-name'))
     logger = logging.getLogger('DocMatcher')
+
+    formater = papis.config.getstring("formater")
+    if formater == "python":
+        doc_format = '{%s[DOC_KEY]}' % (
+            papis.config.getstring('format-doc-name'))
+
+    elif formater == "jinja2":
+        doc_format = '{{%s["DOC_KEY"]}}' % (
+            papis.config.getstring('format-doc-name'))
+
+    else:
+        doc_format = ''
+        logger.error(f"doc_format not implemented for {formater=}")
```

Comment:
```suggestion
        logger.error(f"doc_format not implemented for '%s'", formater)
```
since the `{var=}` syntax is only available in python 3.8 (and papis supports >=3.5).

Reply:
Thank you for the feedback, I wasn't aware of that. Fixed it in the new commit.